tests/emulators/ibmfakebelem_q01/platform.py

```diff
@@ -25,6 +27,14 @@ def create():
     instruments = {"pulse_simulator": pulse_simulator}
     instruments = load_instrument_settings(runcard, instruments)
 
+    # noise_model
+    SNR = 30  # dB
+    NOISE_AMP = np.power(10, -SNR / 20)
+    AWGN = lambda t: np.random.normal(loc=0, scale=NOISE_AMP, size=len(t)) * 3e4
+
+    if instruments["pulse_simulator"].readout_simulator_config is not None:
+        instruments["pulse_simulator"].readout_simulator_config["noise_model"] = AWGN
```

Comment:
So `instruments["pulse_simulator"].readout_simulator_config` is referencing `instruments["pulse_simulator"].settings["readout_simulator_config"]`. This is a problem as we call a method to dump the instrument settings at https://github.com/qiboteam/qibolab/blob/0.1/src/qibolab/serialize.py#L205, which the `PulseSimulator` returns its settings at https://github.com/dexonoir/qibolab/blob/emulator-readout/src/qibolab/instruments/emulator/pulse_simulator.py#L113. By adding a function to this dictionary, the serialization to JSON will fail. Platform dumping is called by Qibocal, so this implementation will break Qibocal as-is.

I would suggest to rework how the noise model is assigned or make a deepcopy of the config to remove the reference (which is the quick and dirty way to do it).